semantic-conventions/src/tests/data/yaml/errors/events/no_event_type.yaml

```diff
@@ -0,0 +1,20 @@
+groups:
+  - id: has.events
+    type: span
+    prefix: has.events
+    events:
+      - random.event
+    brief: example
+    attributes:
+      - id: flag
+        type: boolean
+        brief: An attribute.
+
+  - id: random.event
+    type: span
+    prefix: random
+    brief: example
+    attributes:
+        - id: flag
+          type: boolean
+          brief: An attribute.
```

Comment:
nit
```suggestion
      - id: flag
        type: boolean
        brief: An attribute.
```

Reply:
Thanks! Maybe we should add yamllint to this repo too.